lib/perl/Genome/Model.pm

```diff
@@ -471,6 +471,8 @@ sub __display_name__ {
 sub create {
     my $class = shift;
 
+    no warnings 'redefine';
+    local *__errors__ = sub { () };
```

Comment:
When the software transaction commits at the end of create(), it checks whether all the objects changed within are valid (same as when the main Context commits).  The Model object it's created might not yet be completely valid, but it's valid enough to pass back to the caller.